test/verify/check-metrics

```diff
@@ -368,6 +368,38 @@ class TestHistoryMetrics(MachineCase):
                 });
             })"""))
 
+            b.logout()
+
+        #
+        # Journal logs
+        #
+
+        # Journal was recorded on Fedora 33 and when trying to use it with older systemd it fails with:
+        # `Journal file /var/log/journal/*/journal.journal uses an unsupported feature, ignoring file.`
+        if self.machine.image != "fedora-33":
```

Comment:
This is a bit dangerous -- it will silently stop working once we drop fedora-33. Can we make it `TEST_OS_DEFAULT`, so that we'll learn once we try and switch to 34? It's not very clean, but it's the best idea that I have to avoid a time bomb.

Alternatively, add a list of OSes where the recording does not work, like debian-stable, ubuntu-2004, rhel-8 and so on. I seriously hope journal files are forward compatible.

Reply:
Good call, this would really be a great candidate to become forgotten. I think list of OSes where it does not work is the best way forward.